packages/nextjs/components/Header.tsx

```diff
@@ -63,6 +65,22 @@ export const Header = () => {
     useCallback(() => setIsDrawerOpen(false), []),
   );
 
+  const { address } = useAccount();
+
+  const { data: isInAllowList } = useScaffoldReadContract({
+    contractName: "BatchRegistry",
+    functionName: "allowList",
+    args: [address],
+  });
+
+  const { data: userContractAddress } = useScaffoldReadContract({
+    contractName: "BatchRegistry",
+    functionName: "yourContractAddress",
+    args: [address],
+  });
+
+  const isCheckedIn = userContractAddress && userContractAddress !== zeroAddress;
```

Comment:
Nitpick: We can shortern this condition to just `const isCheckedIn = userContractAddress !== zeroAddress;`

Reply:
You marked this as solved, but it seems that you didn't push the change @kocaemre

Reply:
> You marked this as solved, but it seems that you didn't push the change @kocaemre

Yes,I forgot push the change. Thank you @carletex